.platform.app.yaml

```diff
@@ -0,0 +1,92 @@
+# This file describes an application. You can have multiple applications
+# in the same project.
+
+# The name of this app. Must be unique within a project.
+name: php
+
+# The toolstack used to build the application.
+type: "php"
```

Comment:
Not sure if that matters here, but what about using `python` instead (we don't have any PHP code to execute)?